### PR TITLE
Meeting notes for 2016 07 22

### PR DESCRIPTION
**Pull request by spnc-omz**
Sorry for the delay on meeting notes. Please review.

### CODE REVIEW

File: 2016-07/july-22.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+* We all agreed on the following list of repository team leads:
+  * JSON5-loader: **gdi2290**
+  * HTML-loader: **hemanth**
+  * Mocha-loader: **tricoder42**
+  * Transform-loader: **minwe**
+  * JShint-loader: **kostasmanionis**
+  * I18n-webpack-plugin: **ecutdavid**
+  * Grunt-webpack **danez**
+  * Karma-webpack: **mikaak**
+  * ESlint-loader: (Obsolete)?
```
Comment (kentcdodds) [inline 2016-07/july-22.md]: Oh really? Interesting. Why is that?

Comment (sokra) [inline 2016-07/july-22.md]: That's wrong. We don't own the eslint-loader and it's not obsolete.

Comment (TheLarkInn) [inline 2016-07/july-22.md]: Changing now. That was a misnote thank you for spotting this @kentcdodds

Comment (spnc-omz) [inline 2016-07/july-22.md]: Corrected, thank you.

**Pull request merged**